babs/cli.py

```diff
@@ -1138,13 +1138,86 @@ def check_df_job_specific(df, job_status_path_abs, type_session, which_function)
     return df
 
 
+def _parse_sync_code():
+    """Create and configure the argument parser for the `babs sync-code` command.
+
+    Returns
+    -------
+    argparse.ArgumentParser
+    """
+    parser = argparse.ArgumentParser(
+        description='Save and push code changes to input dataset.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        '--project_root',
+        '--project-root',
+        help=(
+            'Absolute path to the root of BABS project. '
+            "For example, '/path/to/my_BABS_project/' "
+            '(default is current working directory).'
+        ),
+        default=os.getcwd(),
+    )
+    parser.add_argument(
+        '--commit_message',
+        '--commit-message',
+        help='Commit message for datalad save',
+        default='[babs] sync code changes',
+    )
+
+    return parser
+
+
+def _enter_sync_code(argv=None):
```

Comment:
Since this is a new command I don't think we need to support a `babs-sync-code`.